srtcore/group.h

```diff
@@ -264,7 +264,7 @@ class CUDTGroup
     /// @param[out] w_curseq       Group's current sequence number (either -1 or the value used already for other links)
     /// @param[out] w_parallel     Parallel link container (will be filled inside this function)
     /// @param[out] w_final_stat   Status to be reported by this function eventually
-    /// @param[out] w_max_sendable_weight Maximum weight value of sendable links
+    /// @param[out] w_maxActiveWight Maximum weight value of active links
```

Comment:
I think you better name it `w_maxweight`. That's clear enough and the names shouldn't be long in the parameter list unnecessarily.

Reply:
I agree `w_maxActiveWight` is a bit long, but `w_maxweight` would be a bit confusing without looking at the comment.

Reply:
Wight also contains a kinda typo